crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -2163,7 +2162,10 @@ mod tests {
         sync::atomic::{AtomicUsize, Ordering},
     };
 
-    #[expect(dead_code)]
+    #[expect(
+        dead_code,
+        reason = "This struct is used to test how `Drop` behavior works in regards to SparseSet storage, and as such is solely a wrapper around `DropCk` to make it use the SparseSet storage. Because of this, the inner field is intentionally never read."
```

Comment:
This feels like a duplicated drop check test (the other instance being in `crates/bevy_ecs/src/lib.rs`, with the structs `DropCk` and `DropCkSparse`)...